docs/en/index.md

```diff
@@ -10,6 +10,10 @@ The Fluent module provides multi-language translation and localisation for your
 
 If you were looking for localisation of hardcoded strings, go to the [i18n docs](https://docs.silverstripe.org/en/developer_guides/i18n/)
 
+## Installation
+
+Use composer to install fluent, see [installation](./01_installation.md).
```

Comment:
Why is this one done differently to all the others? We should be putting a composer command here and renaming the old installation page (or deleting if no longer relevant)

Reply:
Fair enough